draft/draft-ietf-netconf-https-notif.xml

```diff
@@ -306,7 +306,9 @@ Content-Length: nnn
         </t>
       </section>
       <section title="Response">
-        <t>The response should be "204 (No Content)".</t>
+        <t>The response on success should be "204 (No Content)". In
+        case of corrupted or malformed event, the response should be
+        one of HTTP error responses.</t>
```

Comment:
OLD: one of HTTP error response
NEW: an appropriate HTTP error response

Thoughts?